javascript/app-discover/src/components/DeciParams.tsx

```diff
@@ -55,9 +55,11 @@ export const DeciParams: React.FC = memo(function DeciParams() {
 						label={x.label}
 						key={x.inputProps?.name}
 						labelPosition={Position.top}
-						value={deciParams?.training_options?.[
-							x.inputProps?.name as keyof DECITrainingOptions
-						]?.toString()}
+						value={
+							deciParams?.training_options?.[
+								x.inputProps?.name as keyof DECITrainingOptions
+							]?.toString() || x.defaultValue
```

Comment:
just confirming that an empty string is invalid, and that if the input string is empty we'll use the default value

Reply:
Yes, that it Chris!